### PR TITLE
fixes ai job listing in cargo

### LINKED ISSUE #18168: AI Order Manifest
(WEB REPORT BY: sirskeleto REMOTE: 172.93.109.202:7777)
# Revision
3899bd88a590cda04ab655cdfe53d05db9af16ad

# Description
Any orders placed by the AI show its job title as Cyborg instead of AI.
# Steps to Reproduce
1. Order from cargo as an AI
# What you Expected
YOURNAMEHERE, AI
# What Actually Happened
YOURNAMEHERE, Cyborg

### PR DESCRIPTION
**Pull request by PublicEnemyMaiPai**
closes #18168
makes it so when and AI orders supplies their job is listed as AI
:cl:
 * rscadd: Adds AI as job listing for AI when ordering supplies from cargo

### CODE REVIEW

File: code/game/machinery/computer/cargo.dm [hunk 1]
```diff
@@ -18,6 +18,9 @@ For vending packs, see vending_packs.dm*/
 	else if(issilicon(user))
```
Comment (ShiftyRail) [inline code/game/machinery/computer/cargo.dm]: Did you test it ?
`issilicon(user)` should pass if the user is an AI - the check you added will never trigger.
Change the order of the checks.

**Comment by DamianX** [comment]
You messed up [the auto-close thingy](https://help.github.com/articles/closing-issues-using-keywords/).

> Adds AI as job listing for AI when ordering supplies from cargo

what?

File: code/game/machinery/computer/cargo.dm (hunk 1 above)
Comment (PublicEnemyMaiPai) [reply code/game/machinery/computer/cargo.dm]: fukkkk

**Comment by PublicEnemyMaiPai** [comment]
the cargo order form lists the ai's job as Cyborg, now it will list as AI. also what did i do wrong?

**Comment by DamianX** [comment]
I find that wording confusing.

Oh, and there's an extra # between "Closes" and the issue number.

**Pull request merged**